.github/workflows/run-tests.yml

```diff
@@ -0,0 +1,41 @@
+name: run-tests
+
+on: [push, pull_request]
+
+jobs:
+  test:
+    runs-on: ubuntu-18.04
```

Comment:
Only thing here is do our servers use ubuntu 18.04?

Reply:
No we're using 16.04 but it's becoming deprecated and also it's a public package. Doesn't really matter much.